cmd/argoexec/commands/wait.go

```diff
@@ -31,7 +35,21 @@ func waitContainer(ctx context.Context) error {
 	stats.StartStatsTicker(5 * time.Minute)
 
 	defer func() {
-		if err := wfExecutor.KillSidecars(ctx); err != nil {
+		// Killing sidecar containers
+		retryCnt := 0
```

Comment:
We currently don't have retry counter in other places. It might be better to add later in all places where we are retrying.